kbs/Cargo.toml

```diff
@@ -33,6 +33,9 @@ aliyun = ["kms/aliyun"]
 # Use pkcs11 resource backend to store secrets in an HSM
 pkcs11 = ["cryptoki"]
 
+# Use Nebula CA to provide credentials for nodes (pods) to join a Nebula overlay network
+nebula-ca = []
```

Comment:
maybe we should have a convention for features that enable plugins. we could call this `nebula-ca-plugin` for instance

Reply:
I agree and I like `nebula-ca-plugin`